### PR TITLE
Update/correct the definition of RestOfWorld

### PR DESCRIPTION
**Pull request by shtukas**
We recently introduced the `SubscriptionLocalisation` library ( https://github.com/guardian/price-migration-engine/pull/1133 )

Here, we extend / correct the definition of `RestOfWorld` by adding the GBP case. Tests have been added accordingly.

### CODE REVIEW

**Review by graham228221 — APPROVED** [review]
There is a bit of an edge case where a non-standard country name is used. For example "GB" instead of United Kingdom, or "USA" instead of "United States". Would be tricky to account for all of the variations and I think they'll be quite rare so I think it's ok to ignore (but there are definitely a few in the system).

**Comment by shtukas** [comment]
@graham228221 That's a good point. I checked and displayed the list of all countries appearing in all accounts and they seem to follow a standard format. But yes, it would be preferable to normalise the data and use ISO codes for examples.

File: lambda/src/test/scala/pricemigrationengine/libs/SubscriptionLocalisationTest.scala [hunk 1]
```diff
@@ -4,21 +4,28 @@ import pricemigrationengine.Fixtures
...
-// val subscription = Fixtures.subscriptionFromJson("libs/SubscriptionIntrospection2025/subscription1/subscription.json")
-// val account = Fixtures.accountFromJson("libs/SubscriptionIntrospection2025/subscription1/account.json")
-// val invoicePreview = Fixtures.invoiceListFromJson("libs/SubscriptionIntrospection2025/subscription1/invoice-preview.json")
-// val catalogue = Fixtures.productCatalogueFromJson("libs/SubscriptionIntrospection2025/subscription1/catalogue.json")
+// Subscription 2 is ROW (USD variant), USD is an address in the `France`
+// val subscription = Fixtures.subscriptionFromJson("libs/SubscriptionLocalisation/subscription2/subscription.json")
+// val account = Fixtures.accountFromJson("libs/SubscriptionLocalisation/subscription2/account.json")
+// val invoicePreview = Fixtures.invoiceListFromJson("libs/SubscriptionLocalisation/subscription2/invoice-preview.json")
+
+// Subscription 3 is ROW (GBP variant), USD is an address in the `France`
```
[the remark this answers is not in the document]
Comment (shtukas) [reply lambda/src/test/scala/pricemigrationengine/libs/SubscriptionLocalisationTest.scala]: https://github.com/guardian/price-migration-engine/pull/1151/commits/ce503f53a685fe1978943160eb0744be6b43dde3
